docusaurus/sidebars-ios.json

```diff
@@ -113,6 +113,7 @@
         "guides/customize-message-actions",
         "guides/push-notifications",
         "guides/offline-support",
+        "guides/working-with-user",
```

Comment:
Was this not on the section? 🤔 The rest of the documentation might be outdated then 🤔

Reply:
I found this because I was able to got the page on the current implementation when searching if there was something related to log out in the docs and to see where to add this docs properly, and was able to open it from the search, but the left side bar wasn't showing so I noticed this wasn't added and it appeared again when added to the side-bars JSON. I remember you had a similar issue with the Extra Data docs if I can remember well?

Reply:
In my case the guide was new and created by me, and it was accidentally deleted by a SwiftUI Docs PR. In this case, every document that is not referenced in the sections means it is an incomplete document and should not be used. 

We should move this documentation to here: https://getstream.io/chat/docs/sdk/ios/uikit/getting-started after the "connect user" section, we should create a "Disconnect vs logout" section. This should be the best place for this 👍